README.md

```diff
@@ -62,12 +62,13 @@ be up to 5x slower with some workloads. Please pass
 "-DCMAKE_BUILD_TYPE=RelWithDebInfo" to do_cmake.sh to create a non-debug
 release.)
 
-(Note: `ninja` alone will use only one CPU thread, this could take a while. use
-the `-j` option to use more threads. Something like `ninja -j$(nproc)` would be
-a good start.
+(Note: the number of jobs used by `ninja` is derived from the the number of
+CPU cores of the building host if unspecified. Use the `-j` option to limit
+the job number if the build jobs are running out of memory. On average, each
```

Comment:
The `On average` got duplicated.

```suggestion
the job number if the build jobs are running out of memory. On average, each
 job takes around 2.5GiB memory.
```

Reply:
@rzarzynski thank you for your review! fixed and repushed.